ultra/evaluate.py

```diff
@@ -156,15 +159,9 @@ def evaluate(
         type=str,
         default="easy",
     )
-    parser.add_argument(
-        "--policy",
-        help="Policies available : [ppo, sac, ddpg, dqn, bdqn]",
-        type=str,
-        default="sac",
-    )
     parser.add_argument("--models", default="models/", help="Directory to saved models")
     parser.add_argument(
-        "--episodes", help="Number of training episodes", type=int, default=200
+        "--episodes", help="Number of training episodes", type=int, default=5
```

Comment:
I think we can leave this the same for now. If not, we should update the README as well.

Reply:
oops, I forgot to revert it back to its original. value I will do it right now